lib/front_matter.js

```diff
@@ -171,24 +171,17 @@ function stringifyYAML(obj, options) {
 function stringifyJSON(obj) {
   return JSON.stringify(obj, null, '  ')
     // Remove indention
-    .replace(/\n {2}/g, function() {
-      return '\n';
-    })
+    .replace(/\n {2}/g, () => '\n')
     // Remove prefixing and trailing braces
     .replace(/^{\n|}$/g, '');
 }
 
 function doubleDigit(num) {
-  return num < 10 ? '0' + num : num;
+  return num < 10 ? `0${num}` : num;
 }
 
 function formatDate(date) {
-  return date.getFullYear() + '-' +
-    doubleDigit(date.getMonth() + 1) + '-' +
-    doubleDigit(date.getDate()) + ' ' +
-    doubleDigit(date.getHours()) + ':' +
-    doubleDigit(date.getMinutes()) + ':' +
-    doubleDigit(date.getSeconds());
+  return `${date.getFullYear()}-${doubleDigit(date.getMonth() + 1)}-${doubleDigit(date.getDate())} ${doubleDigit(date.getHours())}:${doubleDigit(date.getMinutes())}:${doubleDigit(date.getSeconds())}`;
```

Comment:
thank @YoshinoriN !
LGTM!
However, I think that it is a bit long here as a line.
Since there is no problem with the operation itself, I will leave a comment while Merge.